db/migrations/20230705160301_travel_settings_migration.php

```diff
@@ -0,0 +1,37 @@
+<?php
+declare(strict_types=1);
+
+use Phinx\Migration\AbstractMigration;
+
+final class TravelSettingsMigration extends AbstractMigration
+{
+    /**
+     * Migrate Up.
+     */
+    public function up(): void
+    {
+        // Modify user_settings table
+        $this->execute("
+            ALTER TABLE `user_settings`
+                ADD `travel_animation` VARCHAR(50) NOT NULL DEFAULT 'smooth',
+                ADD `travel_grid` VARCHAR(50) NOT NULL DEFAULT 'visible',
```

Comment:
Personally - I would like to drop these two settings. I think we can fix the grid and pick one travel motion to align on